README.md

```diff
@@ -1,173 +1,246 @@
 
-E.g. to deploy the latest build:
+### Install from repository
+
+A couple of `make` targets are provided in this repository to allow installing without OLM:
 
 ```bash
-make deploy
+git clone https://github.com/netobserv/network-observability-operator.git && cd network-observability-operator
+make deploy deploy-loki deploy-grafana
 ```
 
-You must then install the custom resource, e.g.:
+It will deploy the operator in its latest version, with port-forwarded Loki and Grafana.
+
+> Note: the `loki-deploy` script is provided as a quick install path, it is not suitable for production. Please refer to [the official documentation](https://grafana.com/docs/loki/latest/) for a clean install.
+
+To deploy the monitoring pipeline, this `make` target installs a `FlowCollector` with default values:
 
 ```bash
-kubectl apply -f ./config/samples/flows_v1alpha1_flowcollector.yaml
+make deploy-sample-cr
 ```
 
-## Build / push / deploy
+Alternatively, you can [grab and edit](./config/samples/flows_v1alpha1_flowcollector.yaml) this config before installing it.
 
-The repository `quay.io/netobserv/network-observability-operator` is only writable by the CI, so you need to use another repository (such as your own one) if you want to use your own build.
+> Note: if you are not using [OVN-Kubernetes](https://github.com/ovn-org/ovn-kubernetes/) CNI, we recommend using `ebpf` as `spec.agent`, rather than `ipfix` (unless you know what you do). NetObserv will automatically configure OVN-Kubernetes for IPFIX exports, but cannot do such with other CNIs.
 
-For instance, to build from a pull-request, checkout that PR (e.g. using github CLI or `git fetch upstream pull/99/head:pr-99 && git checkout pr-99` (replace `99` with the PR ID)), then run:
+You can still edit the `FlowCollector` after it's installed: the operator will take care about reconciling everything with the updated configuration:
 
 ```bash
-IMG="quay.io/youraccount/network-observability-operator:v0.0.1" make image-build image-push deploy
+kubectl edit flowcollector cluster
 ```
 
-Note, the default image pull policy is `IfNotPresent`, so if you previously deployed the operator on a cluster and then create another build with the same image name/tag, it won't be pulled in the cluster registry. So you need either to provide a different image name/tag for every build, or modify [manager.yaml](./config/manager/manager.yaml) to set `imagePullPolicy: Always`, then re-deploy.
+#### Install older versions
 
-Then, you can deploy a custom resource, e.g.:
+To deploy a specific version of the operator, you need to switch to the related git branch, then add a `VERSION` env to the above make command, e.g:
 
 ```bash
-kubectl apply -f ./config/samples/flows_v1alpha1_flowcollector.yaml
-
-# or using make
-make deploy-sample-cr
+git checkout 0.1.2
+VERSION=0.1.2 make deploy deploy-loki deploy-grafana
```

Comment:
we cannot blindly take the branch name (because I might be in my own `feature-xyz` branch and still want to use `main` here), so we would have to do something more clever parsing the branch name .. but not impossible.
I'd rather not mix up several things in this PR and focus strictly on doc at the moment :)

Reply:
ok